src/styles/components/temp-placeholders.styl

```diff
@@ -65,3 +65,19 @@
 .flex-column
   display: flex
   flex-direction: row
+
+.grommet
```

Comment:
I don't think this'll be necessary with the patch to 0.4.3 since I fixed the font-size for the footer.